src/Arcus.Testing.Logging/Arcus.Testing.Logging.csproj

```diff
@@ -19,7 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="Guard.Net" Version="1.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.8" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
```

Comment:
Changing it like this, feels like a breaking change. But not sure if it technically qualifies as a breaking change for us.

Reply:
I don't think it is a problem to use 3.1.8 in .NEt 6 projects.   I don't know if you can use 6.0.0 in .NET Core projects ...

Reply:
Ok, I reverted back to 3.1.8. Should we support both, for now? I mean .NET 6 and .NET Core? Maybe that's a good thing, for now.

Reply:
If possible, yes, I think we should support .NET 6 & .NET Core

Reply:
Ok, sounds like a plan 👍 . We can complete this one now, then.

Reply:
Seems like I'm not a code owner or something, as my review is not enough to complete this. 
I can also not assign someone to this PR

Reply:
Aha, I'll change that.